pkg/storage/sqlstorage/migrations_test.go

```diff
@@ -155,8 +155,7 @@ var postMigrate = map[string]func(t *testing.T, store *sqlstorage.Store){
 			return
 		}
 	},
-	"1": func(t *testing.T, store *sqlstorage.Store) {},
-	"8": func(t *testing.T, store *sqlstorage.Store) {
+	"10": func(t *testing.T, store *sqlstorage.Store) {
 
 		count, err := store.CountTransactions(context.Background(), storage.TransactionsQuery{})
 		if !assert.NoError(t, err) {
```

Comment:
I think that a cleanup of this file is necessary to remove this kind of unnecessary lines of code

Reply:
Completely agree

Reply:
Deleting this file is easier haha